src/transformers/modeling_outputs.py

```diff
@@ -46,6 +46,25 @@ class BaseModelOutput(ModelOutput):
     attentions: Optional[Tuple[torch.FloatTensor]] = None
 
 
+@dataclass
+class BaseModelOutputWithNoAttention(ModelOutput):
+    """
+    Base class for model's outputs, with potential hidden states and attentions.
```

Comment:
```suggestion
    Base class for model's outputs, with potential hidden states.
```
This class doesn't have attention :p

Reply:
thanks :)